packages/protobuf/src/messages-schema.ts

```diff
@@ -1385,7 +1393,7 @@ export const EnumFailureType = Type.Enum(FailureType);
 export type Failure = Static<typeof Failure>;
 export const Failure = Type.Object(
     {
-        code: Type.Optional(EnumFailureType),
+        code: Type.Optional(Type.KeyOfEnum(FailureType)),
```

Comment:
hmm this is a generated file 🤔 was this change added manually?

Reply:
no, from `update:schema` script

but (!!) i also had some (from my pov unwanted) changes when i used `update:protobuf`, is it possible that some recent changes was not commited?
```
export type Failure = {
- code?: keyof typeof FailureType;
+  code?: FailureType;

Reply:
could be, I was changnig this and might have made this mistake - changing generated file. will check on it!